cluster_traffic_capture/demo_haproxy.py

```diff
@@ -47,6 +47,38 @@ def main():
     if verbose:
         logging.root.setLevel(logging.DEBUG)
 
+    # =================================================================================================================
+    # Pull Necessary State
+    # =================================================================================================================
+    print("Pulling AWS credentials from ENV variables...")
```

Comment:
not a blocker, but this would be a nice thing to put behind a flag. We'll likely still want to run this without providing aws credentials (and therefore not offloading to S3).

Reply:
Trying not to invest too heavily in this script as a lot of it will be either deleted or re-written when we move away from HAProxy.  For the time being, you just need to have the ENV variables defined; they don't need to be valid. In that case, the container will spin up but CloudWatch Agent won't be able to upload.